changelog/7437.bugfix.rst

```diff
@@ -0,0 +1 @@
+When calling :meth:`~reproject_to` along with both context managers :meth:`~propagate_with_solar_surface` and :meth:`~assume_spherical_screen` now raises a warning.
```

Comment:
The docbuild is failing due to:

```
<unknown>:59: WARNING: py:meth reference target not found: reproject_to
<unknown>:59: WARNING: py:meth reference target not found: propagate_with_solar_surface
<unknown>:59: WARNING: py:meth reference target not found: assume_spherical_screen
```

These references need to be changed, I think it would be easier to have the full path to the function.

Reply:
Ah missed that, Thanks :+1: